### PR TITLE
PR: Fix backticks, add bullets and minor clarifications to IPython Console doc

### PR DESCRIPTION
**Pull request by mmagnuski**
Last try, I got far too many conflicts when rebasing my 3.x.

### Fixes
in one place the backticks did not render well.

### Clarifies
another change (first in order) is just for clarity.

### CODE REVIEW

File: doc/ipythonconsole.rst [hunk 1]
```diff
@@ -117,7 +117,7 @@ Even better, Spyder also loads the ``%autoreload`` magic by default into any ker
...
 Or, if you're analyzing data step by step using your own custom libraries, you can easily add or tweak a function in the latter and see the results reflected in the former, all without the overhead of reloading the data and re-running your whole script to restore your session to the same point.
 
-UMR is enabled by default, and will do its work automatically without user intervention, although it will provide you with a red ``Reloaded modules:`` message in the console listing the files it has refreshed when it activates. If desired, you can turned it and the message on and off, and prevent specific modules from being reloaded, under :menuselection:`Preferences --> Python interpreter --> User Module Reloader (UMR)`.
+UMR is enabled by default, and will do its work automatically without user intervention, although it will provide you with a red ``Reloaded modules:`` message in the console listing the files it has refreshed when it activates. If desired, you can turn it and the message on or off, and prevent specific modules from being reloaded, under :menuselection:`Preferences --> Python interpreter --> User Module Reloader (UMR)`.
```
Comment (CAM-Gerlach) [inline doc/ipythonconsole.rst]: While you're at it, can you also fix the missing line break between the two sentences in this line? Thanks.

**Comment by CAM-Gerlach** [comment]
Thanks @mmagnuski , looks good except for one minor additional issue with the line you changed, and then I'll approve and merge.

In the future, can you just force-push the changes to the same remote branch as I explained (with ``git push -f``) so its less extra work and clutter with three duplicate PRs? Thanks.

**Comment by mmagnuski** [comment]
> In the future, can you just force-push the changes to the same remote branch as I explained (with git push -f) so its less extra work and clutter with three duplicate PRs? Thanks.

Sorry for the clutter, I tried doing that but git told me that everything is up to date:
```
C:\src\spyder-docs (ipyconsole_docs_fix1 -> origin)
λ git push origin ipyconsole_docs_fix -f
Everything up-to-date
```
but the list of commits was the same on the previous PR so I decided to open yet another one.
I didn't have such a bad git-day as today for a long time. :)

File: doc/ipythonconsole.rst (hunk 1 above)
Comment (mmagnuski) [reply doc/ipythonconsole.rst]: Sure, done.

**Comment by mmagnuski** [comment]
Thanks for your help and patience @CAM-Gerlach!
🚀

**Pull request merged**